compiler/env/OMRCPU.cpp

```diff
@@ -88,3 +105,42 @@ OMR::CPU::initializeByHostQuery()
 #endif
 
    }
+
+bool
+CPU::supportsFeature(uint32_t feature)
+   {
+   TR_ASSERT_FATAL((1<<(feature%32)) & _featureMasks[feature/32] == 0, 
+                   "The %lu feature needs to be added to the enabledFeatures array"
```

Comment:
It would be safer to use `OMR_PRIu32` here
```suggestion
                   "The " OMR_PRIu32 " feature needs to be added to the enabledFeatures array"
```